### PR TITLE
feat: pass project name to quickstart

### PR DESCRIPTION
**Pull request by Eileen-Yu**
resolve https://github.com/diagridio/issues/issues/4445

the execution program receive project name passed in as the env var, and create resource based on that

go with https://github.com/diagridio/cli/pull/535

### CODE REVIEW

**Review by Eileen-Yu — COMMENTED** [review]
kv/javascript works fine, next is to copy/paste these changes to all types

File: pubsub/python/publisher/main.py [hunk 1]
```diff
@@ -29,4 +29,8 @@ async def publish_orders(order: Order):
             return {'success': True}
         except grpc.RpcError as err:
             logging.error(
-                f"Error occurred while publishing order: {err.code()}")
\ No newline at end of file
+                f"Error occurred while publishing order: {err.code()}")
+
+@app.get('/')
```
Comment (Eileen-Yu) [inline pubsub/python/publisher/main.py:34]: the program will periodically call the root endpoint to check health

File: keyvalue/csharp/scaffold.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
         app['workDir'] = '.'
         app['command'] = ['dotnet', 'run']
         app['env']['PORT'] = 5001
-        app['env']['ASPNETCORE_URLS'] = 'http://localhost:5001'
+        app['env']['ASPNETCORE_URLS'] = 'http://0.0.0.0:5001'
```
Comment (Eileen-Yu) [inline keyvalue/csharp/scaffold.py:13]: fix for run it in container

**Review by Eileen-Yu — COMMENTED** [review]
readyyy

File: invocation/csharp/entrypoint.sh [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+fi
+
+echo "Using config file: $CONFIG_FILE"
+
+if [ -n "$QUICKSTART_PROJECT_NAME" ]; then
+  export PROJECT_NAME="$QUICKSTART_PROJECT_NAME"
+else
+  echo "Error: QUICKSTART_PROJECT_NAME is not set."
+  exit 1
+fi
```
Comment (sicoyle) [inline invocation/csharp/entrypoint.sh:16]: let's combine these if conditionals and put the exports together pls for all `entrypoint.sh` files.

**Review by sicoyle — APPROVED** [review]
pls address the one comment and then this is g2g. Nice job!

File: invocation/csharp/entrypoint.sh (hunk 3 above)
Comment (Eileen-Yu) [reply invocation/csharp/entrypoint.sh]: nice catch, updated 😄

**Comment by Eileen-Yu** [comment]
gonna merge this bc CLI need it :)

**Pull request merged**